demo/lightning/text-based/distributed/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Base Image: Start with a slim Python image that already includes Python3.
```

Comment:
Either in this PR or a follow up, can you add a README?

Reply:
Yes that will be added in the followup